spec/integration/kwargs_spec.rb

```diff
@@ -23,8 +23,8 @@ module Test
 
       attr_reader :block
 
-      def initialize(*args, &block)
-        super(*args)
+      def initialize(*args, **kwargs, &block)
+        super(*args, **kwargs)
```

Comment:
Would it make things simpler if this just called `super(**kwargs)`? I think it's absolutely reasonable to expect people to do this. If you a) include auto-inject module b) override constructor then you are on your own. The whole point of using dry-auto-inject is to not have to define constructors. It's allowed to do that of course, but it should be an exception.

Reply:
@solnic 
I don't think it would make any difference. This already works with 2.6, 2.7 and 3.0. Since specs emulate client code - it can be either way.